logproc.go

```diff
@@ -113,18 +113,23 @@ func logProcess(in chan *logData, out chan *logMetrics) {
 		}
 
 		log.Debugln(*data.line)
-		output := strings.Split(*data.line, " - ")
+		var output []string
+		output = strings.Split(*data.line, " - ")
+		if len(output) < 2 {
+			output = strings.Split(*data.line, " app[heroku-redis]: ")
+		}
 		if len(output) < 2 {
 			continue
 		}
```

Comment:
This looks as duplicity. That cause worse readability. Make change naming for getting redis parts and not reuse `output` variable will be better for this case. 

@w-vi what do you think?

Reply:
I see your point 👍 let me try with an explicit variable to get the redis bits!